src/arch/x86/lib/cpu_performance/cpu_info.c

```diff
@@ -39,10 +47,10 @@ static void set_cpu_freq(struct cpu_info *info) {
 					: "=g"(reg[0]), "=g"(reg[1])
 					:
 					: "%eax", "%ebx", "%ecx", "%edx");
-
-	info->base_freq = reg[0];
-	info->max_freq = reg[1];
-
+	
+	
```

Comment:
Please, remove second blank line; we're avoiding placing more than one blank line in a row https://github.com/embox/embox/wiki/Code-Style

Reply:
Sorry for this! Will fix.